sdk/src/driftClient.ts

```diff
@@ -3286,23 +3331,45 @@ export class DriftClient {
 		}
 		const cancelOrderIx = await this.getCancelOrderByUserIdIx(userOrderId);
 
+		const newOrderTypeString = JSON.stringify(newOrderType);
```

Comment:
since these anchor generated types are pretty weird, we generally use the `isVariant`, `getVariant` helpers from this sdk instead of comparing json string